sdk/hybridcompute/Azure.ResourceManager.HybridCompute/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0 (2024-11-27)
```

Comment:
The REST API version is preview so can't release a stable version of SDK

Reply:
fixed